src/index.html

```diff
@@ -1,14 +1,23 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Stop watch</title>
-  <link rel="stylesheet" href="./styles/main.scss">
-</head>
-<body>
-  <h1>Stop watch</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Stop watch</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body>
+    <div class="stop-watch">
+      <div class="stop-watch__line stop-watch__line--top"></div>
+      <div class="stop-watch__line stop-watch__line--bottom"></div>
+      <div class="stop-watch__line stop-watch__line--left"></div>
+      <div class="stop-watch__line stop-watch__line--right"></div>
+      <div class="stop-watch__center"></div>
```

Comment:
top, bottom, left, and right should be settled using CSS, you don't need it in HTML as classes